docs/intellij-quickstart.md

```diff
@@ -15,63 +10,78 @@ git clone https://github.com/hashgraph/hedera-services.git
-  <li>Populate your <i>hedera-node/data/config</i> directory with 
-  dev-appropriate versions of the <i>application.properties</i>,
-  <i>api-permission.properties</i>, and <i>node.properties</i> 
-  files that are used to bootstrap a new Hedera Services network.
-  (To read more about how properties are sourced in Services, 
-   please see [here](./services-configuration.md).)
-</ol>
+- [Android IntelliJ plugin](https://plugins.jetbrains.com/plugin/22989-android)
+  adds additional code navigation capabilities for Dagger2 annotated interfaces.
+- [JMH IntelliJ plugin](https://plugins.jetbrains.com/plugin/7529-jmh-java-microbenchmark-harness)
+  allows running selected JMH benchmarks directly from the IDE.
 
-## Starting a local single-node network
+## Configure the JDK used by Gradle
 
-Now browse to `com.hedera.node.app.ServicesMain`. Its
-`main` method starts a single node network of Hedera Service by
-calling `com.swirlds.platform.Browser#main`, which is the
-entrypoint to bootstrap the Platform app named by the
-[_config.txt_](../hedera-node/config.txt) in the working
-directory.
+The project is imported as a Gradle project. Before you can use all features reliably, make sure that Gradle is
+started with the JDK we use in the project, which currently is:
+**Eclipse Temurin, 21.0.1**
 
-Run `ServicesMain#main` with an IntelliJ configuration whose working
-directory is the _hedera-node/_ directory of your clone of this repo:
+You can use IntelliJ to download the JDK if you do not have it installed.
 
 <p>
-    <img src="./assets/node-config.png" height="518" width="800" />
+    <img src="./assets/gradle-jdk.png" />
 </p>
 
-You can also run from Gradle below:
+
+## Reload Project with Gradle
+
+After everything is configured, and everytime you change something in the project setup, you should press the
+**Reload All Gradle project** in IntelliJ.
+
+Changes to the project setup include,
+- Changing `build.gradle.kts` files
+- Changing dependencies in `src/main/java/module-info.java` files
+- Changing build configuration in `gradle/plugins/src/main/kotlin`
 
 <p>
-    <img src="./assets/gradle-run.png" height="214" width="415" />
+    <img src="./assets/gradle-reload.png" />
 </p>
 
-You will see a single black pane appear, similar to:
+See [gradle-quickstart](gradle-quickstart.md) for more details on modifying the project setup.
+
+## Build, test, run through Gradle tasks
+
+You can run all tasks described in [gradle-quickstart](gradle-quickstart.md) from the Gradle tool window.
 
 <p>
-    <img src="./assets/node-startup.png" height="150" width="600"/>
+    <img src="./assets/gradle-tasks.png" />
 </p>
 
-This node's name is "Alice" because of [Line 26](../hedera-node/config.txt#L26)
+## Running a services instance or example apps
+
+### Starting a local single-node network
+
+You can start the `ServicesMain` process via the `modrun` Gradle task. You will see an output like the following:
```

Comment:
I have filed a new issue (#13836) that removes `modrun` and just uses `run`. We needed `modrun` because we needed the ability to run both the modular version of services and the mono version of services. But now that we have moved everything to modular code, we can just let `run` be the task.